Vagrantfile

```diff
@@ -1,27 +1,42 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+['vagrant-reload'].each do |plugin|
+  unless Vagrant.has_plugin?(plugin)
+    raise "Vagrant plugin #{plugin} is not installed!"
+  end
+end
+
 Vagrant.configure("2") do |config|
+  config.ssh.forward_agent = true
+
   config.vm.box = "debian/stretch64"
 
   config.vm.synced_folder "src/glb-wireshark-dissector/", "/home/vagrant/.config/wireshark/plugins/glb-wireshark-dissector", type: 'rsync'
 
-  config.vm.provision "shell", inline: <<-SHELL
+  config.vm.provision :shell, inline: <<-SHELL
+    echo 'deb http://ftp.debian.org/debian stretch-backports main' >/etc/apt/sources.list.d/backports.list
     apt-get update
-    DEBIAN_FRONTEND=noninteractive apt-get install -y tcpdump net-tools tshark build-essential libxtables-dev linux-headers-$(uname -r) python-pip jq bird curl libsystemd-dev
+
+    DEBIAN_FRONTEND=noninteractive apt-get -y dist-upgrade
```

Comment:
Just gets it all up to date without backports since the Vagrant image may not be up to date IIRC, I believe the backports are still deactivated by default (only updated to when `-t stretch-backports`).